### ISSUE TITLE
[Question] Which library should I be using? Confusing ReadMe file

### ISSUE THREAD

**Issue by ThiagoRangel7** [Seeed-Studio/PN532#89]
I'm starting to work with [Arduino NFC Module V3 (by Elechouse)](http://www.elechouse.com/elechouse/index.php?main_page=product_info&cPath=90_93&products_id=2242) aiming to connect it to Android phones, but I'm really confused, because there are a lot of different Arduino NFC libraries out there.

Should I use this library ([Seeed-Studio/PN532](https://github.com/Seeed-Studio/PN532)) or the other one called [Seeed_Arduino_NFC](https://github.com/Seeed-Studio/Seeed_Arduino_NFC)?

Also, I noticed that the ReadMe file says that the "easy way" to get started is by downloading this library and [Don's NDEF library](https://github.com/don/NDEF), but, at the same time, there's the Seeed-Studio's *Seeed_Arduino_NFC* (mentioned above) which is forked from *Don's NDEF* and seems to be more up-to-date. Is the ReadMe file outdated?

These ReadMe files are really confusing, since each one points to other libraries, but none of them mentions the new *Seeed_Arduino_NFC*, which seems to include Don's and this library in just one library.

**TL; DR:**
Which Arduino NFC Library should I use? Which one is more up-to-date?
* [Seeed-Studio/PN532](https://github.com/Seeed-Studio/PN532)
* [Seeed-Studio/Seeed_Arduino_NFC](https://github.com/Seeed-Studio/Seeed_Arduino_NFC)
* [Don/NDEF](https://github.com/don/NDEF) (seems outdated - latest commit from 1y ago, and *Seeed_Arduino_NFC* is forked from this, although ReadMe file says that we should download it)
* [elechouse/PN532](https://github.com/elechouse/PN532) (seems outdated - latest commit 1y ago and is forked from *Seeed-Studio/PN532*)
* [Adafruit-PN532](https://github.com/adafruit/Adafruit-PN532) (seems bad)

**Comment by Pillar1989** [comment]
@ThiagoRangel7  
Thank you for your attention to PN532 software. The software on the market may be a little misleading, but we only maintain the following two software：
* [Seeed-Studio/PN532](https://github.com/Seeed-Studio/PN532)
* [Seeed-Studio/Seeed_Arduino_NFC](https://github.com/Seeed-Studio/Seeed_Arduino_NFC)

The PN532 library is basic software that I hope will be useful to non-arduino users.

Seeed_Arduino_NFC is the software we developed for Arduino users.  It comes from Don. Examples can help Arduino get started with NFC application quickly
